Frontend-Development/Contribute/Readme.md

```diff
@@ -27,3 +27,4 @@
 - [The Palisadoes Foundation](https://summerofcode.withgoogle.com/archive/2021/organizations/4835102856577024)
 - [Wikimedia Foundation](https://summerofcode.withgoogle.com/archive/2021/organizations/5270263742070784)
 - [xfce](https://summerofcode.withgoogle.com/archive/2021/organizations/5706869816950784)
+- [React play](https://reactplay.io/)
```

Comment:
Please arrange the community name in alphabetical order

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient